Pages/FoodCustomer/Cart.cshtml

```diff
@@ -47,7 +47,23 @@
                     @Html.DisplayFor(modelItem => food.Food.ApplicationUser.RestaurantName)
                 </td>
                 <td>
-                    @Html.DisplayFor(modelItem => food.Food.Price)
+                    @{
+                        if (food.Food.CurrentPrice.IsDiscount)
+                        {
```

Comment:
@ symbol could be used before 'if', like @if (...........) { } because extra lines are not needed at all

Reply:
Yes, will be formated correctly